diff -urNp -x '*.orig' nessus-libraries.org/configure.in nessus-libraries/configure.in
--- nessus-libraries.org/configure.in	2005-10-27 15:29:18.000000000 +0200
+++ nessus-libraries/configure.in	2021-09-29 22:27:43.770919561 +0200
@@ -73,13 +73,15 @@ test -z "$GCC" || CWARN="-Wall"
 
 dnl Check for extra libraries
 
-AC_HAVE_LIBRARY(resolv, LIBS="-lresolv $LIBS")
+LIBRESOLV=""
+AC_CHECK_FUNC(res_query, , [AC_CHECK_FUNC(__res_query, , [AC_CHECK_LIB(resolv, res_query, [LIBRESOLV="-lresolv"], [AC_CHECK_LIB(resolv, __res_query, [LIBRESOLV="-lresolv"])])])])
+AC_SUBST(LIBRESOLV)
 
 # These libraries break stuff under IRIX
 if  test "`uname`" != "IRIX" ;
 then
- AC_HAVE_LIBRARY(socket, [socket_lib="-lsocket";LIBS="-lsocket $LIBS";])
- AC_HAVE_LIBRARY(nsl, [nsl_lib="-lnsl";LIBS="-lnsl $LIBS";])
+ AC_CHECK_FUNC(socket, , [AC_CHECK_LIB(socket, socket, [socket_lib="-lsocket";LIBS="-lsocket $LIBS";])])
+ AC_CHECK_FUNC(inet_aton, , [AC_CHECK_LIB(nsl, inet_aton, [nsl_lib="-lnsl";LIBS="-lnsl $LIBS";])])
 fi
 
 
@@ -381,9 +383,10 @@ fi
 
 # disable openpty() upon request, only
 if test x$USE_OPENPTY != xno -a x$USE_PTMX != xyes ; then
-  AC_CHECK_LIB(util,openpty,LIBS="-lutil $LIBS")
+  LIBUTIL=""
+  AC_CHECK_LIB(util,openpty,LIBUTIL="-lutil")
   save_LDFLAGS=$LDFLAGS
-  LDFLAGS="$LDFLAGS $LIBS"
+  LDFLAGS="$LDFLAGS $LIBS $LIBUTIL"
   AC_CHECK_FUNCS(openpty,FOUND_OPENPTY=yes)
   LDFLAGS=$save_LDFLAGS
   # undef the request for openpty as it is not available
@@ -632,7 +635,7 @@ dnl Define several paths
 AC_SYS_LONG_FILE_NAMES
 
 
-EXTRA=$LIBS
+EXTRA="$LIBS $LIBRESOLV"
 
 case "$host" in
   *-darwin*)
diff -urNp -x '*.orig' nessus-libraries.org/libhosts_gatherer/Makefile nessus-libraries/libhosts_gatherer/Makefile
--- nessus-libraries.org/libhosts_gatherer/Makefile	2003-04-07 04:06:49.000000000 +0200
+++ nessus-libraries/libhosts_gatherer/Makefile	2021-09-29 22:27:43.774252887 +0200
@@ -22,7 +22,7 @@ install : libhosts_gatherer.la
 	$(LIBTOOL) --finish $(DESTDIR)${libdir}
 	
 libhosts_gatherer.la : $(OBJS)
-	$(LIBTOOL) $(CC) -o libhosts_gatherer.la $(LO_OBJS) -rpath ${libdir} \
+	$(LIBTOOL) $(CC) -o libhosts_gatherer.la $(LO_OBJS) $(LIBRESOLV) -rpath ${libdir} \
 	-version-info ${PACKAGE_VERSION}
 	        
 hg_utils.o : hg_utils.h hosts_gatherer.h hg_utils.c
diff -urNp -x '*.orig' nessus-libraries.org/nessus.tmpl.in nessus-libraries/nessus.tmpl.in
--- nessus-libraries.org/nessus.tmpl.in	2003-02-17 15:52:01.000000000 +0100
+++ nessus-libraries/nessus.tmpl.in	2021-09-29 22:27:43.770919561 +0200
@@ -69,6 +69,7 @@ LDFLAGS=
 # Your C compiler
 CC=@CC@
 LIBS=@LIBS@
+LIBRESOLV=@LIBRESOLV@
 LIBTOOL=@LIBTOOL@
 RANLIB=@RANLIB@
 
