diff -urN nessus-libraries.orig/Makefile nessus-libraries/Makefile
--- nessus-libraries.orig/Makefile	Mon Sep 23 20:10:24 2002
+++ nessus-libraries/Makefile	Mon Sep 23 20:43:49 2002
@@ -34,8 +34,9 @@
 	-cd libpcap-nessus && ${MAKE}
 	 
 pcap-install:
-	test -d ${prefix} || ${INSTALL_DIR} -m 755 ${prefix}
-	test -d ${prefix}/lib || ${INSTALL_DIR} -m 755 ${prefix}/lib
+	test -d $(DESTDIR)${prefix} || ${INSTALL_DIR} -m 755 $(DESTDIR)${prefix}
+	test -d $(DESTDIR)${prefix}/lib || \
+		${INSTALL_DIR} -m 755 $(DESTDIR)${prefix}/lib
 	-cd libpcap-nessus && ${MAKE} install
 	 	 
 pcap-clean :
@@ -45,24 +46,26 @@
 	-cd libpcap-nessus && ${MAKE} distclean
 	
 install : $(PCAP_INSTALL)
-	test -d ${prefix} || ${INSTALL_DIR} -m 755 ${prefix}
-	test -d ${includedir}/nessus || ${INSTALL_DIR} -m 755 ${includedir}/nessus
+	test -d $(DESTDIR)${prefix} || ${INSTALL_DIR} -m 755 $(DESTDIR)${prefix}
+	test -d $(DESTDIR)${includedir}/nessus || \
+		${INSTALL_DIR} -m 755 $(DESTDIR)${includedir}/nessus
 	cd libnessus && ${MAKE} install
 	cd libhosts_gatherer && ${MAKE} install
 
 
-	$(INSTALL) -m 0444 include/includes.h ${includedir}/nessus
-	$(INSTALL) -m 0444 include/libnessus.h ${includedir}/nessus
-	$(INSTALL) -m 0444 include/harglists.h ${includedir}/nessus
-	$(INSTALL) -m 0444 include/libvers.h   ${includedir}/nessus
-	$(INSTALL) -m 0444 include/getopt.h    ${includedir}/nessus
-	test -d ${bindir} || ${INSTALL_DIR} -m 755 ${bindir}
-	test -d ${sbindir} || ${INSTALL_DIR} -m 755 ${sbindir}
-	$(INSTALL) -m 0755 nessus-config ${bindir}/nessus-config
-	$(INSTALL) -m 0755 uninstall-nessus ${sbindir}/uninstall-nessus
-	test -d ${mandir} || ${INSTALL_DIR} -m 755 ${mandir}
-	test -d ${mandir}/man1 || ${INSTALL_DIR} -m 755 ${mandir}/man1
-	$(INSTALL) -m 0644 nessus-config.1 ${mandir}/man1
+	$(INSTALL) -m 0444 include/includes.h $(DESTDIR)${includedir}/nessus
+	$(INSTALL) -m 0444 include/libnessus.h $(DESTDIR)${includedir}/nessus
+	$(INSTALL) -m 0444 include/harglists.h $(DESTDIR)${includedir}/nessus
+	$(INSTALL) -m 0444 include/libvers.h   $(DESTDIR)${includedir}/nessus
+	$(INSTALL) -m 0444 include/getopt.h    $(DESTDIR)${includedir}/nessus
+	test -d $(DESTDIR)${bindir} || ${INSTALL_DIR} -m 755 $(DESTDIR)${bindir}
+	test -d $(DESTDIR)${sbindir} || ${INSTALL_DIR} -m 755 $(DESTDIR)${sbindir}
+	$(INSTALL) -m 0755 nessus-config $(DESTDIR)${bindir}/nessus-config
+	$(INSTALL) -m 0755 uninstall-nessus $(DESTDIR)${sbindir}/uninstall-nessus
+	test -d $(DESTDIR)${mandir} || ${INSTALL_DIR} -m 755 $(DESTDIR)${mandir}
+	test -d $(DESTDIR)${mandir}/man1 || \
+		${INSTALL_DIR} -m 755 $(DESTDIR)${mandir}/man1
+	$(INSTALL) -m 0644 nessus-config.1 $(DESTDIR)${mandir}/man1
 
 	@echo
 	@echo ' --------------------------------------------------------------'
diff -urN nessus-libraries.orig/libhosts_gatherer/Makefile nessus-libraries/libhosts_gatherer/Makefile
--- nessus-libraries.orig/libhosts_gatherer/Makefile	Mon Sep 23 20:10:24 2002
+++ nessus-libraries/libhosts_gatherer/Makefile	Mon Sep 23 20:44:26 2002
@@ -17,9 +17,10 @@
 	$(COMPILE) -static test.c -o test -L.libs -lhosts_gatherer $(LIBS)
 
 install : libhosts_gatherer.la
-	test -d ${libdir} || $(INSTALL_DIR) -m 755 ${libdir}
-	$(LIBTOOL) --mode install $(INSTALL) libhosts_gatherer.la ${libdir} 
-	$(LIBTOOL) --finish ${libdir}
+	test -d $(DESTDIR)${libdir} || $(INSTALL_DIR) -m 755 $(DESTDIR)${libdir}
+	$(LIBTOOL) --mode install $(INSTALL) libhosts_gatherer.la \
+		$(DESTDIR)${libdir} 
+	$(LIBTOOL) --finish $(DESTDIR)${libdir}
 	
 libhosts_gatherer.la : $(OBJS)
 	$(LIBTOOL) $(CC) -o libhosts_gatherer.la $(LO_OBJS) -rpath ${libdir} \
diff -urN nessus-libraries.orig/libnessus/Makefile nessus-libraries/libnessus/Makefile
--- nessus-libraries.orig/libnessus/Makefile	Mon Sep 23 20:10:24 2002
+++ nessus-libraries/libnessus/Makefile	Mon Sep 23 20:44:51 2002
@@ -69,9 +69,10 @@
 	-version-info ${PACKAGE_VERSION}
 
 install : libnessus.la
-	test -d ${libdir} || $(INSTALL_DIR) -m 755 ${libdir}
-	$(LIBTOOL) --mode install $(INSTALL) libnessus.la ${libdir}/libnessus.la
-	$(LIBTOOL) --finish ${libdir}
+	test -d $(DESTDIR)${libdir} || $(INSTALL_DIR) -m 755 $(DESTDIR)${libdir}
+	$(LIBTOOL) --mode install $(INSTALL) libnessus.la \
+		$(DESTDIR)${libdir}/libnessus.la
+	$(LIBTOOL) --finish $(DESTDIR)${libdir}
 
 distclean : clean
 
diff -urN nessus-libraries.orig/libpcap-nessus/Makefile.in nessus-libraries/libpcap-nessus/Makefile.in
--- nessus-libraries.orig/libpcap-nessus/Makefile.in	Mon Sep 23 20:10:24 2002
+++ nessus-libraries/libpcap-nessus/Makefile.in	Mon Sep 23 20:46:10 2002
@@ -118,21 +118,25 @@
 install-:
 
 install-libpcap-nessus.la :
-	$(LIBTOOL) --mode install $(INSTALL) libpcap-nessus.la ${exec_prefix}/lib 
-	$(LIBTOOL) --finish ${exec_prefix}/lib
+	$(LIBTOOL) --mode install $(INSTALL) libpcap-nessus.la \
+		$(DESTDIR)${exec_prefix}/lib 
+	$(LIBTOOL) --finish $(DESTDIR)${exec_prefix}/lib
 
 install-incl: force
-	test -d ${includedir}/nessus || \
-		$(INSTALL) -m 755 -d ${includedir}/nessus
-	test -d ${includedir}/nessus/net || \
-		$(INSTALL) -m 755 -d ${includedir}/nessus/net
-	$(INSTALL) -m 444 $(srcdir)/pcap.h ${includedir}/nessus/pcap.h
-	$(INSTALL) -m 444 $(srcdir)/pcap-namedb.h \
-		${includedir}/nessus/pcap-namedb.h
-	$(INSTALL) -m 444 $(srcdir)/net/bpf.h ${includedir}/nessus/net/bpf.h
+	test -d $(DESTDIR)${includedir}/nessus || \
+		$(INSTALL) -m 755 -d $(DESTDIR)${includedir}/nessus
+	test -d $(DESTDIR)${includedir}/nessus/net || \
+		$(INSTALL) -m 755 -d $(DESTDIR)${includedir}/nessus/net
+	$(INSTALL) -m 444 $(DESTDIR)$(srcdir)/pcap.h \
+		$(DESTDIR)${includedir}/nessus/pcap.h
+	$(INSTALL) -m 444 $(DESTDIR)$(srcdir)/pcap-namedb.h \
+		$(DESTDIR)${includedir}/nessus/pcap-namedb.h
+	$(INSTALL) -m 444 $(DESTDIR)$(srcdir)/net/bpf.h \
+		$(DESTDIR)${includedir}/nessus/net/bpf.h
 
 install-man: force
-	$(INSTALL) -m 444 -o bin -g bin $(srcdir)/pcap.3 ${mandir}/man3/pcap.3
+	$(INSTALL) -m 444 -o bin -g bin $(DESTDIR)$(srcdir)/pcap.3 \
+		$(DESTDIR)${mandir}/man3/pcap.3
 
 clean:
 	@rm -f $(CLEANFILES) net/net
