--- nessus-libraries.orig/Makefile	Tue Jan 21 14:25:01 2003
+++ nessus-libraries/Makefile	Thu Feb 27 18:09:48 2003
@@ -34,8 +34,8 @@
 	-cd libpcap-nessus && ${MAKE}
 
 pcap-install:
-	test -d ${prefix} || ${INSTALL_DIR} -m 755 $(DESTDIR)${prefix}
-	test -d ${prefix}/lib || ${INSTALL_DIR} -m 755 $(DESTDIR)${prefix}/lib
+	test -d $(DESTDIR)${prefix} || ${INSTALL_DIR} -m 755 $(DESTDIR)${prefix}
+	test -d $(DESTDIR)${prefix}/lib || ${INSTALL_DIR} -m 755 $(DESTDIR)${prefix}/lib
 	-cd libpcap-nessus && ${MAKE} install
 
 pcap-clean :
